src/components/attendee/NavOverlay/NavOverlay.tsx

```diff
@@ -55,18 +55,18 @@ export const NavOverlay: React.FC<navOverlayProps> = ({
   };
 
   return (
-    <div className={SN.NavOverlay}>
-      <div className={SN.NavOverlay__close} onClick={onClose}>
+    <div className={CN.navOverlay}>
+      <div className={CN.navOverlay__close} onClick={onClose}>
         Close
-        <span className={classNames("NavOverlay__close-icon", SN.closeIcon)} />
+        <span className={cn("NavOverlay__close-icon", CN.closeIcon)} />
       </div>
-      <div className={SN.NavOverlay__container}>
-        <div className={SN.NavOverlay__navigation}>
+      <div className={CN.navOverlay__container}>
```

Comment:
Having `__` in the field names could trip up linters.  camelCase would mean we replace these kinds of classes with `CN.navOverlayContainer`

```suggestion
      <div className={CN.navOverlayContainer}>
```

If these naming instances are the only notes I have, I will approve the PR so it can be merged sooner and make a new one to iron out inconsistencies.

Reply:
[0dc4703](https://github.com/sparkletown/sparkle/pull/2813/commits/0dc470385bb9fc6a84f4a1e50a08c986d4edd8a7)